doc/finance.md

```diff
@@ -0,0 +1,12 @@
+# Accounting and finances
+
+## Where can I find the budget?
+
+See [the Board Google Drive](#google-drive) for our budget information.
+It is in the {kbd}`Governing Board` -> {kbd}`Budget` folder.
+
+## Where can I find monthly summaries of the budget?
+
+The **Monthly Summaries folder** contains an updated budget with the latest information about expenses.
+It is in [the Board Google Drive](#google-drive).
+Find the {kbd}`Governing Board` -> {kbd}`Budget` -> {kbd}`Monthly summaries` folder.
```

Comment:
The downside of not being able to link to these folders directly is that these kinds of instructions will often become outdated.